src/Files.App/Data/Items/ListedItem.cs

```diff
@@ -326,6 +329,41 @@ public ObservableCollection<FileProperty> ItemProperties
 			set => SetProperty(ref itemProperties, value);
 		}
 
+		public string DimensionsDisplay
```

Comment:
I discovered this is fetching as soon as the location is opened

Reply:
This also causes #15202

Reply:
I'll be commenting this out because the feature is low priority, but it'll be nice to add back once these issues are resolved.

Reply:
@hishitetsu The first implementer said this way is the fastest however speaking of this should we use WinRT API to retrieve the dimentions of images?

Reply:
I think the tooltip value can be empty at first, since the tooltip value is updated each time the pointer is entered.
https://github.com/files-community/Files/blob/0923cf0c03d17f48c97804f5fbe609a97185b576/src/Files.App/Views/Layouts/GridLayoutPage.xaml.cs#L675-L680

> @hishitetsu The first implementer said this way is the fastest however speaking of this should we use WinRT API to retrieve the dimentions of images?

I don't know if using the WinRT API will solve any problems.